src/commands/dev/index.ts

```diff
@@ -98,6 +98,15 @@ class UpProcessManager {
       });
     });
 
+    this.server.on('error', (e: any) => {
+      if (e.code === 'EADDRINUSE' && this.server) {
```

Comment:
What happens if this errors twice? This seems like it could be an infinite loop.

Reply:
Hmm yep, not sure if there's a circumstance that could make that happen, but theoretically we could get into an infinite delete/recreate/error/repeat loop. I'll update this to hard-fail if we already tried to fix this error and then got it again